src/com/itschoolsession4/homework/ex1/Tema1.java

```diff
@@ -0,0 +1,49 @@
+package com.itschoolsession4.homework.ex1;
+
+
+import java.util.Scanner;
+
+public class  Tema1
+{
+    public static void main(String[] args) {
+        //Create a Scanner object for user input
+        Scanner scanner =new Scanner( System.in);
+
+        // Prompt the user to enter two numbers
+        System.out.println("Enter the first number: ");
+        double num1 = scanner.nextDouble();
+
+        System.out.println("Enter the second number: ");
+        double num2 =scanner.nextDouble();
+
+        //Perform calculations
+        double addition = num1 + num2;
+        double subtraction = num1 -num2;
+        double multiplication = num1 * num2;
+        double division =(num2 != 0)    ? num1/num2 : Double.NaN; //Avoid division by zero

```

Comment:
you are checking only `num2` to be different than 0.. but what if `num1` is 0? same for below `modulus`.